parlai/core/torch_agent.py

```diff
@@ -1693,6 +1871,17 @@ def batchify(self, obs_batch, sort=False):
                     x_lens, xs, x_lens, valid_inds, exs, descending=True
                 )
 
+        # TIME_STEPS
+        ts = t_lens = None
+        if any(ex.get('timestep_vec') is not None for ex in exs):
+            _ts = [ex.get('timestep_vec', self.EMPTY) for ex in exs]
+            ts, t_lens = self._pad_tensor(_ts)
+            if sort:
```

Comment:
Is it possible that this gets affected by the `sort` change in [line 1869]( https://github.com/facebookresearch/ParlAI/blob/8271c8520d11514a39ef38bbe1b1ee9ed9b2d3e2/parlai/core/torch_agent.py#L1869)

Reply:
I am not even sure we need this sort here. The sort on batching is important for text, but may not be relevant for the timestamps, which shouldn't have wildly varying lengths.

Reply:
Yea I think it's gonna mess up the `sort` change earlier in text, and I agree it's really not necessary to have sort by length for timesteps. I've gone ahead and removed the sort for timesteps.